kubernetes/readme.md

```diff
@@ -0,0 +1,18 @@
+#Kubernete Setup
+
+```
+#create secrets
+kubectl create secret generic todo-secrets --from-literal=DB_URL=xxxxx
+
+# apply
+kubectl apply -f deployment.yaml
+kubectl apply -f service.yaml
+
+
+#get container
+kubectl get pods   -o wide
+kubectl get pod module-14-7d4d477ddb-p6m9q  --template='{{(index (index .spec.containers 0).ports 0).containerPort}}{{"\n"}}'
+
+#port forward
+kubectl port-forward module-14-7d4d477ddb-p6m9q 5000:5000
```

Comment:
Here it looks like you're directly port forwarding to the pod rather than the service - this works but we generally want to use the service since that's meant to be our route into the cluster. Note that you've shifted the service port to 7080, so you'll want something like:
`kubectl port-forward service/module-14 5000:7080` so that 5000 on your host -> 7080 in the cluster (which then routes it onto one of your pods' port 5000s again!)